### PR TITLE
release-1.1.0

### PR DESCRIPTION
**Pull request by wenchajun**
Signed-off-by: chengdehao <dehaocheng@kubesphere.io>

### Which issue(s) this PR fixes:

Fixes #

### Does this PR introduced a user-facing change?

```release-note

```

### Additional documentation, usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by benjaminhuo** [comment]
We need to include a fix for the below problem in 1.1.0 :
https://fluent-all.slack.com/archives/C02642TE9HR/p1654700442484299

**Comment by benjaminhuo** [comment]
@wenchajun You can rebase the master branch with this PR

**Comment by wenchajun** [comment]
> @wenchajun You can rebase the master branch with this PR to include the new change and update release note accordingly

done

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
+## 1.0.2 / 2022-06-09
+
+### Features
+- Fluent-bit integrate OpenSearch plugin (#298)
```
Comment (benjaminhuo) [inline CHANGELOG.md:4]:
```suggestion
- Add OpenSearch plugin for Fluent Bit (#298)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+- Add support for user defined annotations (#313)
+- Fluentd integrate opensearch plugin (#324)
```
Comment (benjaminhuo) [inline CHANGELOG.md:6]:
```suggestion
- Add OpenSearch plugin for Fluentd(#324)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,18 @@
+## 1.0.2 / 2022-06-09
+
+### Features
+- Fluent-bit integrate OpenSearch plugin (#298)
+- Add support for user defined annotations (#313)
```
Comment (benjaminhuo) [inline CHANGELOG.md:5]:
```suggestion
- Support custom annotations (#313)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,18 @@
...
+- Fluentd integrate opensearch plugin (#324)
+- Add helm template and docs about opensearch output plugin (#329)
```
Comment (benjaminhuo) [inline CHANGELOG.md:7]:
```suggestion
- Add helm & docs for OpenSearch plugin (#329)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,18 @@
...
+
+### ENHANCEMENT
+- Move some content to fluent operator walkthrough (#290)
```
Comment (benjaminhuo) [inline CHANGELOG.md:10]:
```suggestion
- Move some docs to fluent operator walkthrough (#290) 
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,18 @@
...
+
+### Features
+- Fluent-bit integrate OpenSearch plugin (#298)
+- Add support for user defined annotations (#313)
+- Fluentd integrate opensearch plugin (#324)
+- Add helm template and docs about opensearch output plugin (#329)
+
+### ENHANCEMENT
+- Move some content to fluent operator walkthrough (#290) 
+- Organize and update documents (#291 #292 #293 #303 #314)
```
Comment (benjaminhuo) [inline CHANGELOG.md:11]:
```suggestion
- Docs refactoring (#291 #292 #293 #303 #314)
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,18 @@
...
+- Use "string" Type for Key_does_not_exist (#317)
```
Comment (benjaminhuo) [inline CHANGELOG.md:12]:
```suggestion
```
This has been reverted

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,18 @@
...
+- Update go version (#316)
+- Integration with systemd plugin (#323)
```
Comment (benjaminhuo) [inline CHANGELOG.md:14]:
```suggestion
- Use a single systemd input plugin for various components (#323)
```

**Review by benjaminhuo — APPROVED** [review]

**Review by benjaminhuo — APPROVED** [review]

**Pull request merged**